lib/app/modules/home/views/home_view.dart

```diff
@@ -1432,3 +1443,48 @@ class HomeView extends GetView<HomeController> {
     );
   }
 }
+
+Future<bool> showDeleteAlarmConfirmationPopupOnSwipe(
+  BuildContext context,
+) async {
+  // Return true if user confirms deletion, false if canceled
+
+  return await showDialog(
```

Comment:
You'll have to use GetX for dialog since that is what we've been using so far for all our dialogs.